docs/ref/clvm.md

```diff
@@ -742,7 +745,249 @@ cost = 806
+(a (q 2 (q 2 (i 11 (q 2 (i (= 5 (point_add 11 (pubkey_for_exp (sha256 11 (a 6 (c 2 (c 23 ()))))))) (q 2 23 47) (q 8)) 1) (q 4 (c 4 (c 5 (c (a 6 (c 2 (c 23 ()))) ()))) (a 23 47))) 1) (c (q 50 2 (i (l 5) (q 11 (q . 2) (a 6 (c 2 (c 9 ()))) (a 6 (c 2 (c 13 ())))) (q 11 (q . 1) 5)) 1) 1)) (c (q . 0x848f09f98800442737684dd76071f25a0bd100b51e727aabafeddb062dbc3d2b3ac64bc87f084a6d16e4e89e1417de14) 1))
+```
+
+```bash
+(venv) chia-blockchain $ opd ff80ffff01ffff3dffa023f61666150d2a467ee7b81a77954c93255d65c0c43108f1bb14ac420fd59c428080ff8080
+(() (q (61 0x23f61666150d2a467ee7b81a77954c93255d65c0c43108f1bb14ac420fd59c42)) ())
+```
+
+```bash
+(venv) chia-blockchain $ brun -c '(a (q 2 (q 2 (i 11 (q 2 (i (= 5 (point_add 11 (pubkey_for_exp (sha256 11 (a 6 (c 2 (c 23 ()))))))) (q 2 23 47) (q 8)) 1) (q 4 (c 4 (c 5 (c (a 6 (c 2 (c 23 ()))) ()))) (a 23 47))) 1) (c (q 50 2 (i (l 5) (q 11 (q . 2) (a 6 (c 2 (c 9 ()))) (a 6 (c 2 (c 13 ())))) (q 11 (q . 1) 5)) 1) 1)) (c (q . 0x848f09f98800442737684dd76071f25a0bd100b51e727aabafeddb062dbc3d2b3ac64bc87f084a6d16e4e89e1417de14) 1))' '(() (q (61 0x23f61666150d2a467ee7b81a77954c93255d65c0c43108f1bb14ac420fd59c42)) ())'
+cost = 15032
+((50 0x848f09f98800442737684dd76071f25a0bd100b51e727aabafeddb062dbc3d2b3ac64bc87f084a6d16e4e89e1417de14 0x03db13c4e422e5eea98463c02b2c15994b620e0a45aa2db6f7785d3ba28f46cf) (61 0x23f61666150d2a467ee7b81a77954c93255d65c0c43108f1bb14ac420fd59c42))
+```
+
+The CLVM cost for this coin spend was 15,032.
+
+We still need to calculate this program's size cost.
+  * The serialized puzzle is 582 characters, or 291 bytes
+  * The serialized solution is 94 characters, or 47 bytes
+  * The sum of these two is 338 bytes. With a cost of 12,000 per byte, the program's size cost is 4,056,000
+
+Let's add up the costs to get the total cost for this transaction:
+* First coin
+    * CLVM:             39,652
+    * AGG_SIG_ME:    1,200,000
+    * CREATE_COIN 1: 1,800,000
+    * CREATE_COIN 2: 1,800,000 
+    * Program size:  5,160,000
+* Second coin
+    * CLVM:             15,032
+    * AGG_SIG_ME:    1,200,000
+    * Program size:  4,056,000
+* Total:            15,270,684
+
+This is an accurate assessment of the cost for this particular transaction.
+
+Just like before, we have to multiple the cost by five to obtain the miniumum fee. In this case, the result is `5 * 15,270,684 = 76,353,420 mojos`. It's still a good idea to round this fee up to 100 million mojos to ensure that your transaction will skip ahead of the zero-fee transactions in the mempool.
```

Comment:
"multiple" -> "multiply".

The second sentence is a bit unclear to me. 5 mojos per cost *is* the threshold to kick out other transactions from the mempool, why is the rounding necessary?

It seems like you're referring to some inaccuracy in the cost calculation. But we know exactly how the mempool calculates the cost. and iirc, it uses the same mechanism as `cdv`. i.e. it builds a one-transaction block and computes its cost.